erpnext/accounts/page/pos/pos.js

```diff
@@ -1219,11 +1226,11 @@ erpnext.pos.PointOfSale = erpnext.taxes_and_totals.extend({
 		me.bind_delete_event()
 	},
 
-	update_qty: function (item_code, qty) {
+	update_qty: function (item_code, qty, remove_zero_qty_items='') {
```

Comment:
minor change: 
parameters in JS are `undefined` (falsy)  by default. no need for setting a default value here below.